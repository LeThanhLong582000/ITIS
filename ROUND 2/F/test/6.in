100
94144 546200 123671
285967 637037 625911
411272 19591 287991
701852 745318 428781
867378 643764 225542
227780 744390 804804
133998 269539 76426
952540 796561 905007
127993 955588 660771
60477 702991 617577
50220 32377 592652
584893 696325 402798
56551 30435 507575
534973 315732 37166
423080 696156 627864
859851 21293 955385
675056 904403 443615
733527 361807 254665
28584 257768 336077
486870 948343 108944
711360 506118 399402
766459 645741 893768
891523 97237 607115
368857 383107 571685
569315 350976 226151
563165 779699 158900
287410 821543 605905
195517 810553 830392
449100 278549 387591
370910 76725 38059
986558 274656 685378
259661 191179 997933
195749 54809 308636
147918 319778 426971
795741 711015 65028
570579 215019 537680
643051 116247 158724
431729 113968 559163
176677 666041 179945
691211 198908 317193
11784 422508 64446
793549 456643 342341
840693 315843 667126
431275 553406 303849
656968 636439 797242
119914 543903 702128
877110 746080 24873
650703 213401 88849
666610 914404 920267
103201 912752 474873
843380 957560 716685
672896 831948 854876
741685 981135 878140
140296 43406 527408
568902 537333 154425
532050 97396 120519
975933 289602 210575
203469 898893 866355
870515 237619 217105
173135 818693 678352
650663 657748 511632
725063 508086 237200
826989 247678 548212
65476 593201 12717
277234 147415 978201
901645 717330 849926
809956 354039 632300
840108 341266 15546
282047 55680 430513
935715 810263 673796
936642 397316 394811
306639 255065 549920
462359 920525 514073
788108 827812 765569
530227 251288 570215
742432 457003 892164
915013 426946 159327
618453 408480 994648
448675 998083 789285
97881 920709 850365
857587 420523 232715
42568 876235 440025
661825 36707 700553
674636 452593 621675
984832 223755 371797
831438 529271 614637
737952 702603 308864
208393 228906 48258
833173 576568 989099
937994 306763 597757
420237 340916 99291
503009 764052 134752
106154 196111 446532
181684 122856 901550
571522 581990 110999
272303 25110 118182
103157 909896 789443
781405 449836 54362
496634 852989 250237
331429 70569 58750
