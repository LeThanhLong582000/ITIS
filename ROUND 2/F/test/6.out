0
137421
287992
192658
224578
144097
0
530493
0
0
305248
145683
266846
37167
177394
896972
107134
254666
53447
0
302322
507243
607116
278718
222245
0
35886
107678
279071
38060
685379
533208
224788
127556
65029
446620
158725
438462
0
317194
0
339624
595988
90859
408886
139070
24874
88850
336237
0
301253
347912
319345
312149
92997
120520
210576
85466
217106
16397
252274
227089
548213
0
554010
517121
544109
15547
328440
399624
394812
300747
27954
362933
424577
588797
159328
602311
119939
13769
232716
0
662836
421859
371798
458402
172107
13873
622852
597758
89306
0
178288
480189
50266
118183
0
54363
0
58751
